### ISSUE TITLE
Mint needs to be able configure coins amount

### ISSUE THREAD

**Issue by Dreepa** [StrangeLoopGames/EcoIssues#6813]
Currently:
10 coins per item by which it is backed

Problem:
Whatever is in supply will inflate. Counter is a market prices that is adaptive. Like in the real world, supply and demand. If, say you are using gold backed, at some point everyone will be digging gold to make currency. This is when the price of gold would fall.
However, in the game, you cannot reduce the 10 coins given out per gold item.

Suggestion:
Make the price configurable. Even better: Make it possible for the player to create a vote about the price.
Or even add an option for the mint to automatic de-value the item progressively.

**Comment by Huntermeign** [comment]
Mint is a currency made up with a backing. the value itself doesn't matter unless you allow anyone to use the Mint. In other words. The item you choose is simply the cost of the recipe to make the currency and the currency is simply the item that can be traded for goods and services that is akin to credit but accepted by whomever chooses to do so and has the unique option to be given to players via command. Supply and demand is only related to currency if someone chooses to accept a general currency trade off their supply to meet demand. 

tl;dr

don't let anyone and everyone use your mint and focus on the money in circulation rather than the product in circulation.

**Comment by Dreepa** [comment]
No, the backing changes in value as supply rises. A gold backed currency will change value according to the gold price. 

Unless you are talking about "making the coin out of the material" which is also irrelevant, since you can make coins out of fish. So even here, you could say: "We pay with whole fish" or "We pay with fish eyes only", which would attribute to the different value of 1 fish into currency.

**Comment by Huntermeign** [comment]
Lets break this down.

"the backing changes in value as supply rises. A gold backed currency will change value according to the gold price"

The only thing that changes in a backed currency is if you create or destroy said currency. Other than that the currency mint system in the game is for convenience more than it is for tangibility. It can be paid to via command, be put in shops,  and can also be used in contracts. This can also be done with a credit system. Difference being that stores that use a credit line from a creditor that isnt their own has to deal with the idea that someone has unlimited currency. This is why I related it to changing the recipe, because thatch all you're really asking to do.

(and the complexity of managing a commodity's value on the basis of an exchange with supply and demand is probably beyond the scope of the game).

There is nothing complex about supply and demand and definitely not beyond the scope of anything let alone what goes on in a game. Chinese farmers can make WoW gold worth tangible money. This alone shows what real supply and demand is and that it is not at all complex. Items are priced based on demand and no one forces people to price their items a certain way based on the backed currency which is why I said its not related UNLESS they choose to use the currency.  The furthest I can break it down further is to say that it also depends on how much money is in circulation, but unless people print their own money supply and demand and currency are only related in the sense that currency is used as a trade medium. The value of the currency is only relevant based on circulation. NOT on how much supply the minted item is in

"So while the player owning the mint could do that, the 10:1 ratio being a flexible tool could also do that."

"Unless you are talking about "making the coin out of the material" which is also irrelevant, since you can make coins out of fish. So even here, you could say: "We pay with whole fish" or "We pay with fish eyes only", which would attribute to the different value of 1 fish into currency."

See above when I said all you're asking to do is change the recipe.

"As to the mint usage:
People know that 10 currency units will come out of the mint if you use it, so they won't be happy if you make the currency yourself," 

Literally no currency is made unless someone makes it their selves. The only reason currency works is because at some point it is agreed upon as tender. Either because an elected figurehead said this is the currency they would use, or because a merchant made their prices worth it by creating a system where the items they buy are subsidized with a currency they could use to buy other items to meet their demand. As far as Eco is concerned, its has met this twofold and can really only add additional options such as creating trade windows to trade items for items thus creating simple economics. 

"and just give them less for the item than the 10 bucks"

Wait... hold on let me log in.... Okay lets see.

![image](https://user-images.githubusercontent.com/4082137/38181887-984f2c3c-3603-11e8-8437-c89136ff6444.png)

He seems happy doing it. I know I'm happy selling ore at 0.5 a piece when the 0.2 is empty. and its because not anyone can make currency that this can happen. 

"since that's what they would have gained operating the mint themselves."

See above when I said it only works when it is agreed upon.

**Comment by Dreepa** [comment]
I didn't negate your solution. I just said that I think it shouldn't be the only one. I think it should be viable to have the mint public or "open the mint to the public for defined amounts of time".

I do not know why you need to justify your viewpoint like you have found the holy grail. It is really tiresome to be honest. No one debated the stuff you brought up. I just suggested to put the control not at the price level only, but at the mint level too. That is all really.

**Comment by Huntermeign** [comment]
Break down time again.

"I didn't negate your solution"

What do you even mean by this? Its not a solution. It's how the game can be played. What you are suggesting is changing the current game-play. And since this is a game and not actual currency, you need to remember that this is a recipe to create mint as a backing, I get that it hasnt happened in real world situations on a major scale in a while, but despite inflation, the value of whatever you back a currency adjusts despite what is used to create the backing. In other words. ! oz of gold is still 1 oz of gold and while the value of the gold inflates and deflates outside the market, the cost to make a gold coin is always 1 oz.

"I think it should be viable to have the mint public or "open the mint to the public for defined amounts of time"."

It can be with the current setup. Just set up a shop buying gold for however much you want gold to be worth and hide the mint.

"As you have already rightly pointed out, the current solution means the mint is basically controlled by one person who is then owner of the shop that translates the goods into money for price he seems fit (to control the circulation)."

Never said this. I said mint can become tangible by means of people agreeing to make it a currency. If people only want to use one currency then thats fine too. Anyone who can make a mint or buy a mint can mint currency. Not by one person but by one person who makes mint because that was agreed upon by the collective minds. We had 3 collective currencies on our last map. 1 credit and 2 backed. The fact that a credit line became a norm requires a lot of trust in a player to convince a whole server to use it.

"Here is a player situation: Player runs the bank. Backs currency with gold ore. Sees low circulation. Sees high hoarding. Wants to inflate. Announces: "Bank opens to buy gold in 1h"."

There is no 'bank' unless you mean treasury and/or building you designated as a bank which is fine, but you are asking to change a major game mechanics recipe simply because of one situation that will set up an unfavorable mechanic that requires a lot of micromanagement and freedom to players. This is part of the learning process of the game, and one thing to learn from all of this is how to manage money indirectly as well as directly. being able to change the recipe wont do this.

Problem: Can only translate 1 gold ore in 10 currency units, OR set up shop and be seen as fraudster who is cashing in on the difference between the shop's price and mint's recipe (10).

This is not a problem. This is part of the game. We had a guy try to do this on our last map. When we told him we were not going to buy his gold unless he changed the price to something lower he refused at first. Then after we still started minting and he had a bunch of gold he couldnt sell because we weren't at that tech level yet. He eventually sold the gold to 1 currency per bar because it was either that or he made no money. Like I said earlier, while the value of the minted item can be adjusted, it will never change the recipe of the minted item itself.

"People come with carts full of gold. Inflation goes way over intended target. Some people are very rich. Prices rise. Difference between poor and wealthy players massively increases."

Unless a player is running a monopoly then prices rising means demand is rising which is a good thing. If they are rising because of a monopoly then introduce new product into the market to convince them to lower their price, either by selling the exact same items or selling an item that uses the items he/she is selling in a recipe to be at a much lower price and then creating a contract to deliver items at a much cheaper bulk price. No one forces people to buy anything in this game and 30 days is a long time to defeat the meteor, even with incoherent people. Eventually you will want to see several people in each skill.

"Better:
Mint is set to 0,1 per gold. People can use the mint to create money out of their gold ore. Value is low, increase is controllable."

This is in no way better compared to how much damage would be done to the current game mechanics considering everything you've said was an issue has several alternative solutions.

"Add me to steam if you want to debate financial systems. I love doing that. Same nickname."

nty.

**Comment by Dreepa** [comment]
You really are arguing for the sake of arguing. I have stated the use-case multiple times. If you think the game should not be played that way, fine. I think different. Also, backed currency does not mean it is created out of that material, how you make it sound. Nevertheless, I don't see any debate finding place here that has the goal to achieve consensus, but only to be right. You just cannot have other people have their approach to things. Nothing I will say will make you consider my approach as valid, while everything I say will be met with continuous elaboration on why "that is not the way the game is meant to be played". You so get off-tracked in your reply, that I don't even know where to start with responding. 

If you think "That is not a problem", I think it is, as it disables one way of playing the game (which is a public mint).

**Comment by Huntermeign** [comment]
Sorry been busy with work and stuff. Break down time again.

"You really are arguing for the sake of arguing. I have stated the use-case multiple times. If you think the game should not be played that way, fine. I think different."

No. Im arguing because you are proposing something that is game breaking and what you ideally want to do can already be done with credits instead of backing. 

"Also, backed currency does not necessarily mean that it is created out of that material, but that the paper given out entitles the owner to exchange it for the counter-value in gold (which is stored by the banker to protect the gold owner of theft)"|

This you are partly right about since currency in a backing are simply notes saying how much of you have of a generally accepted tender. The said currency being created in this instance though is part of a digital system however and not physical. I cannot go to a mint and turn my currency back into raw material. Even if I could I would still not agree with you unless notes printed and raw material can both be used as tender and can both be simultaneously circulating in the world. Reason being because it reflects what was closer to real life when bank noted currency was created and banks had printed more notes than the amount of material they had and caused inflation. This is why I say you are asking to change the recipe because the item used in mint is forever gone once circulation begins. In fact all you can do to destroy currency is just hold on to money and never spend it.

"This is where the shared reserve started (giving out more entitlements than gold stored, smart banksters)."

No, this is how the audit began as well as inflation because bankers were creating more notes than backing as said above. Neither of these are good things and you are asking for it willingly. Notes created wasn't supposed to change in value. 1 note was MEANT to equal X amount of tender material and that value was never meant to change. Banks notes were smaller and easier to conceal compared to their counterparts which is why they were favored. Due to things such as the removal of a backed currency, the banks printing more notes than currency in-hand, or the banks simply spending all their physical tender, we now have things such as credit, inflation of currency, and federal interference of trade and non-backed currency. Not as smart as you give them credit for. Take away the later part, and what you have is what we are supposed to have. A note system where 1 note is 1/10 of agreed upon tender and nothing about that can change. The game makes it simple to where we cannot lose this money either from theft by making it digital currency. What you want to do is change the value of what the exchange is which is essentially changing the recipe since this is a game and you never see the items again. This is best done with a credit system since it also reflects real life non backed currencies,

"Anyway, since this is a game, it is besides the point, as it is about game-play, not simulating the real world. "

A game partly funded by the DoE with a few awards as well since it is a learning tool at its core. So yes, simulating real world is important.

 "Nevertheless, I don't see any debate finding place here that has the goal to achieve consensus, but only to be right. You have your one and only truth, and there is no room apparently to have other people have their approach to things. Nothing I will say will make you consider my approach as valid, while everything I say will be met with continuous elaboration on why "that is not the way the game is meant to be played". With all due respect, you so get off-tracked in your reply, that I don't even know where to start with responding."

Strawman. You come here with a suggestion of change and thus created a medium where it can be debated and now you are saying the medium is not one to be debated on? No 'debate finding place' is needed to tell you this idea is bad and this tactic you are presenting is to be seen to have no value in what is actually being presented as to either of our ideals. There is no off track. Everything I have said was to break down how your ideas are not a reflection of good game-play or good for the game at all as a simulated learning tool. Saying this though. This IS definitely off track. You don't know where to start responding because you have an idea that can easily be taken apart as a bad idea. If anything you need to present new material to the table if you actually want any part of this to be in the game.

"If you think (about my use-case) "That is not a problem", I think it is, as it disables one way of playing the game (which is a public mint, with whitelisted price control)."

It disables control which, in a system like this, you couldn't possibly have any to begin with because backed materials notes don't change in value like you say it does. Im not going to go store a backed currency at a bank if I were to go to the bank later to find out that my printed money has become useless. You are thinking of now-a-day non-backed currencies and credit which only really work because the non-backed mint printed is in quantities to meet a very very large crowed in which the tender USED to have a backing but was dropped.

Like I said several times. If you really REALLY want a system like this then use a credit system. But it does not reflect the backing system.

So let's agree to disagree, or we will just continue to spam this until no-one will want to read though the walls of text any longer."

No one is spamming. This is a debate on the suggested gameplay. Again, strawman.

**Comment by Huntermeign** [comment]
https://youtu.be/vptO6fsFWdU

**Comment by Dreepa** [comment]
Your breakdown is an opinion disguised. Why do you take the high ground and portraying yourself as an objective judge of truth? Makes discussion with you pointless.

**Comment by Dreepa** [comment]
A debate on game-play? I don't see much debate on game-play, but we are more debating irrelevancies towards the game-play.

If we want to debate game-play, drop the overhead and "paragraph answer to each quote" aka "breakdown rhetoric" which is an opinion in disguise.

State in a few simple sentences, why being allowed to change the mint recipe will break the game.

**Comment by Huntermeign** [comment]
"A debate on game-play? I don't see much debate on game-play, but we are more debating irrelevancies towards the game-play. ("Not side tracked...." lol.... )"

This entire thing entirely based on what you said initially and I break down and respond to everything you say. if there is any sidetracking its on you.

"If we want to debate game-play, drop the overhead and "paragraph answer to each quote" aka "breakdown rhetoric" which is an opinion in disguise (and if I would start doing that, we will soon have an infinite amount of sub-threads of irrelevancies)."

Another strawman. Your post is an opinion under the guise that it adds an extra option to the game. An option already available. You choose to ignore it as well as ignore better methods over what is better in your opinion. That being said this is completely irrelevant to your suggestion. I break it down to simplify my responses so you know which part I am directing to while responding. You speak irrelevancies but you are the one throwing irrelevant arguments into this and calling my opinions just that. All I am doing is relaying to you why this particular idea... is a bad one and should not be implemented because it does not adequately teach backed systems, which the game should do as a learning tool.

"State in a few simple sentences, why being allowed to change the mint recipe will break the game. If you can even do that without going to write another book :)"

Now a condescending tone. Seriously? because you are acting stubborn and hurt that I am telling you why this is not a positive effect on game-play

Also, I already have. Several times with several reasons, but because you either didn't accept the answer and either retorted or replied saying it was a matter of opinion I had to explain further. Here are the simplified answers.

1. This is already an option using credits. Backed systems don't change the amount of tender in circulation. 1 currency = 1/10 agreed upon physical tender. 

2. Allowing anyone to use a mint is an option but not viable because of reason 1 and because bank-notes cannot be traded in for physical tender.

3. It breaks the simple concept of supply and demand by dictating the market prices when it is a natural process.

**Comment by Dreepa** [comment]
Well, maybe you should try to get to the point more quickly then. I am not stubborn. I just work with GD for about 10y now, and digging through walls of text of user feedback is part of my work. Extracting the essence is key. Dabbling leads nowhere, and I can see by your attitude that I wouldn't want to invest too much energy here, so I try to get back on track by condensing it to the main issue. These "so perceived" explanations are just viewpoints. Been there so often, I know it leads nowhere. I could now start to explain why you are wrong again, and you could then again state how cool and superior response was, and I could then respond to that, etc. and eventually we get nowhere. So what is the issue? Definitily not how real world banking works. And also not what you perceive as a valid playstyle. So let's focus on the problem.

1 & 2 are no game-breakers, but a convention definition of how you think it is supposed to work. But anyway, let's talk about all 3.

1:  Credits are a players currency, right? The one he/she has infinite of. Is that the one you talk about? If yes, then those are not backed by an investment cost of items needed to create the currency. Therefore, it is a different use case compared to this suggestion, right?

2: Yes, this is the problem I have raised. You state: Allowing everyone to use a mint is |...| no viable.
This is the use case I stated, and you seem to agree with my statement. 

3. This does not break the game. It merely adds the option for a player to change the price for 10 currency, if they want. If the market asks for the same item for a higher price, people would probably go there. If the market asks for the mint item for a lower price, people will go to the mint. So the mint player could have more impact and regulate inflation. 

BTW: check out https://eco.gamepedia.com/Mint
The screenshots of chats at the bottom of the page (the first one) also shows their problem with inflation. So their solution was to go admin mint only.

**Comment by BlazeGuy2** [comment]
This issue was moved to StrangeLoopGames/EcoSuggestions#167
